DHL/JSON/66da0fe7-7a0b-4e36-b685-8f4d6b06607f.json

```diff
@@ -0,0 +1,531 @@
+[
+  {
+    "ShpInfReqDef": {
+      "Id": "66da0fe7-7a0b-4e36-b685-8f4d6b06607f",
+      "Nm": "Consignee ID required - GR",
+      "Dsc": "Consignee ID required desc",
+      "DrvdFromShpInfReqDef": {},
+      "LedOwnOrgn": {
+        "CtryCd": "GR"
+      },
+      "ApplyAllInd": false,
+      "PrsnDatSenstyClssLvl": "VeryHigh",
+      "QuickGuideEnabledInd": false,
+      "UICmpt": {
+        "CmptNm": "POI",
```

Comment:
Update excel template and update with correct cmptnm

Reply:
Done